mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -2307,6 +2307,17 @@ impl MmCoin for TendermintCoin {
     fn on_disabled(&self) -> Result<(), AbortedError> { AbortableSystem::abort_all(&self.abortable_system) }
 
     fn on_token_deactivated(&self, _ticker: &str) {}
+
+    async fn account_db_id(&self) -> Option<String> {
+        if let Ok(public_key) = self.activation_policy.public_key() {
+            let address_hash = dhash160(&public_key.to_bytes());
+            let address_rmd160_hex = hex::encode(address_hash.as_slice());
+
+            return Some(address_rmd160_hex);
+        };
+
+        None
```

Comment:
We can activate using a different public key for hd wallet too, this goes for pirate too.

Reply:
`get_publickey` returns key for active mode.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/d9d12293bdc6d6f38d8cd75a3c2d9c95b9119a6e/mm2src/coins/tendermint/tendermint_coin.rs#L233-L254

Reply:
Sorry, missed that for some reason. let's fix it for pirate then.

Reply:
done